src/foam/nanos/dig/drivers/DigJsonDriver.js

```diff
@@ -42,6 +42,7 @@ foam.CLASS({
         formatter.setOutputClassNames(true);
         formatter.setOutputDefaultValues(true);
         formatter.setMultiLine(true);
+        formatter.setOutputShortNames(true);
```

Comment:
Why do we want to use short names for DIG?

Reply:
The field was named `externalId` to be generic but the shortname was set to `paymentReference`. We accept shortname on DIG.put call but the response return back the name of the field eg. externalId: "xxx" which could cause confusion.

Reply:
But paymentReference isn't a short name since it is longer. Often short names are only one or two characters long, like 'fn' for 'firstName', so this could also be confusing. This is intended for efficiency when doing FOAM <-> FOAM communications, not for external protocols because it is prone to change and harder to read and debug.

Reply:
seems like short names have been misused for external or alternative field name for DIG, what do you think if we could add `externalName` for this purpose instead of driving it on `shortName`? @kgrgreer

Reply:
ShortNames should be shortNames - like Kevin described. 
Otherwise the model name should be used. 
If we need some other support for DIG, then some other mapping should be used, not externalName. 
Kristina added support for inbound mapping, perhaps it can support reverse mapping. 
But really, what is wrong with externalId? 
And and another extended model overriding names (regular names), is always an option.

Reply:
externalId is fine for me, @adamfox-nanopay can we update the API doc to replace paymentReference with externalId?

Added support for prop names mapping https://github.com/kgrgreer/foam3/pull/3187/commits/366e7cddba348a7fbd097f12f64f7500c4265b20.